README.md

```diff
@@ -97,11 +97,11 @@ start with one of the following option sets.
 
 ### Command Line
 
-`mergerfs -o cache.files=partial,dropcacheonclose=true,category.create=mfs /mnt/hdd0:/mnt/hdd1 /media`
+`mergerfs -o cache.files=partial,dropcacheonclose=true,category.create=mfs /mnt/hdd\*:/mnt/otherhdd:/mnt/rohdd=ro /media`
```

Comment:
Oops, all the more reason to get it up and in people's faces where they can see it right away?

I also made some minor changes so that the potentially devastating *gotcha* with mv is listed where most people are likely to read still before tuning out. (Like I did.)